docs/user-guide/generate-certificates.md

```diff
@@ -113,12 +114,10 @@ After you configure the `zowe.yaml`, use the following procedure to generate the
 
 #>
 ```
-**Detailed explanation of generated output**
-* The `zwe init certificate` command generates a certificate based on `zowe.yaml` values in the `zowe.setup.certificate` section.
-* The certificate values used at runtime are referenced in the `zowe.certificate` section in the `zowe.yaml` file.
-* The command `zwe init certificate -c ./zowe.yaml --update-config` updates the runtime `zowe.certificate` section to reference the generated certificate generated from the `zowe.setup.certificate`.
 
-Open the `zowe.yaml` file to check the references to the newly generated certificate values as shown in the following code snippet.
+The `zwe init certificate` command generates a certificate based on `zowe.yaml` values in the `zowe.setup.certificate` section. The certificate values used at runtime are referenced in the `zowe.certificate` section in the `zowe.yaml` file. The command `zwe init certificate -c ./zowe.yaml --update-config` updates the runtime `zowe.certificate` section to reference the generated certificate generated from the `zowe.setup.certificate`.
```

Comment:
The command `zwe init certificate -c ./zowe.yaml --update-config`

might be more clear -->

The command `zwe init certificate -c <path-to-your-zowe-configuration-yaml> --update-config`

And probably text search replace this whole page for 
`zowe.yaml` 
to 
`Zowe configuration file`

Reply:
Where zowe.yaml is specified in the --update-config command, I have changed this value to <path-to-your-zowe-configuration-yaml>.